### PR TITLE
Migrate to NextJs version

### PR DESCRIPTION
**Pull request by DaniloKachanko**

### CODE REVIEW

File: src/components/menu/menu-switch.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+  return (
+    <a href="#" id="menu-switch" className="menu-switch flex items-center justify-center my-16 mx-32" title="Switch menu" onClick={click}>
+    <svg className="icon path w-32 h-32 fill-dark-super hover:fill-blue" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlnsXlink="http://www.w3.org/1999/xlink">
+        <g className="svg-group">
+          <path className="path lines" d="m0 4v2h24v-2zm0 7v2h24v-2zm0 7v2h24v-2z" />
+          <path className="path cross" d="m4.22 2.81-1.42 1.42 7.78 7.78-7.78 7.78 1.42 1.42 7.78-7.78 7.78 7.78 1.42-1.42-7.78-7.78 7.78-7.78-1.42-1.42-7.78 7.78z" />
+        </g>
+      </svg>
+    </a>
```
Comment (Lowentwickler) [inline src/components/menu/menu-switch.tsx:24]: If menu is here as SVG, why do we have menu.svg file in assets?

File: src/globals.css [hunk 2]
```diff
@@ -0,0 +1,153 @@
+@import url('https://fonts.googleapis.com/css2?family=Jura:wght@300;400;500;600;700&display=swap');
```
[the remark this answers is not in the document]
Comment (Lowentwickler) [reply src/globals.css:1]: use them

File: src/pages/about.tsx [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    </>
+  )
+}
+
+// export const getStaticProps = async () => {
+//     const manager = new ContentManager();
+//     return {
+//       props: manager.getPageProps(manager.page("about")),
+//     };
+//   };
```
Comment (Lowentwickler) [inline src/pages/about.tsx:51]: Clean it up

File: src/pages/contact.tsx [hunk 4]
```diff
@@ -0,0 +1,215 @@
...
+                      marginHeight={0}
+                      marginWidth={0}
+                    ></iframe>
+                  </div>
+                </div>
+
+                <div className="feedback px-48 w-1/2 space-y-8">
+                  {/* <div 
+                   className={classNames("bg-green-alert p-8 text-dark-super font-bold border-t-2 border-green-border","bg-red-alert border-red-border")}
+                   > */}
```
Comment (Lowentwickler) [inline src/pages/contact.tsx]: clean it up

File: src/pages/contact.tsx [hunk 5]
```diff
@@ -0,0 +1,215 @@
...
+  const [feedbackState, setFeedbackState] = useState(FeedbackState.NOT_SENT)
+
+  const getFeedbackStateMessage = (state: FeedbackState) => {
+    switch (state) {
+      case FeedbackState.NOT_SENT:
+        return null
+      case FeedbackState.ERROR:
+        return (
+          <div className="bg-red-alert border-red-border p-8 text-dark-super font-bold border-t-2">
+            Type your message and email
```
Comment (Lowentwickler) [inline src/pages/contact.tsx:40]: We need to handle different errors differently. If backend is down, you will show this message even if everything is filled up.

File: src/pages/contact.tsx [hunk 6]
```diff
@@ -0,0 +1,215 @@
...
+          </div>
+        )
+      case FeedbackState.SENT:
+        return (
+          <div className='bg-green-alert p-8 text-dark-super font-bold border-t-2 border-green-border"'>
+            Succesfull
```
Comment (Lowentwickler) [inline src/pages/contact.tsx:46]: Use message for production website

**Review by Lowentwickler — CHANGES_REQUESTED** [review]

File: content/blog/joining-100-days-of-code.md [hunk 7]
```diff
@@ -5,7 +5,10 @@ draft: false
 description:
 tags: ["leos", "100DaysOfCode"]
 categories: ["news"]
-
+summary: "Some time ago I decided to join the [100DaysOfCode](https://www.100daysofcode.com/) challenge. I 
+wanted to start on June 1 with a new Milestone from the [roadmap](/leos/roadmap/), but I feel some
+lack of focus on the project due to other tasks, so perhaps it is worth starting right now. To get 
+updates follow [me on Twitter](https://twitter.com/IljaDev)."
```
[the remark this answers is not in the document]
Comment (DaniloKachanko) [reply content/blog/joining-100-days-of-code.md:11]: -

File: src/pages/contact.tsx (hunk 5 above)
Comment (DaniloKachanko) [reply src/pages/contact.tsx:40]: I change to "Something went wrong"

**Pull request closed without merging**